### PR TITLE
remove sample codes from zipfile packaging

### PR DESCRIPTION
**Pull request by xiaoyu-work**
## Describe your changes

remove sample codes from zipfile packaging
## Checklist before requesting a review
- [ ] Add unit tests for this change.
- [ ] Make sure all tests can pass.
- [ ] Update documents if necessary.
- [ ] Lint and apply fixes to your code by running `lintrunner -a`
- [ ] Is this a user-facing change? If yes, give a description of this change to be included in the release notes.
- [ ] Is this PR including examples changes? If yes, please remember to update [example documentation](https://github.com/microsoft/Olive/blob/main/docs/source/examples.md) in a follow-up PR.

### CODE REVIEW

File: docs/source/features/packaging_output_models.md [hunk 1]
```diff
@@ -21,9 +17,6 @@ Zipfile packaging will generate a ZIP file which includes 3 folders: `CandidateM
 
 Each `BestCandidateModel` folder will include model file/folder. The folder also includes a json file which includes the Olive Pass run history configurations since input model, a json file with performance metrics and a json file for inference settings for the candidate model if the candidate model is an ONNX model.
 
-#### SampleCode
-Olive will only provide sample codes for ONNX model. Sample code supports 3 different programming languages: `C++`, `C#` and `Python`. And a code snippet introducing how to use Olive output artifacts to inference candidate model with recommended inference configurations.
```
Comment (devang-ml) [inline docs/source/features/packaging_output_models.md:25]: If OrtPerfTuning pass was run, then we need a way to let users know the recommended inference configurations.

Comment (devang-ml) [reply docs/source/features/packaging_output_models.md:25]: Would writing the recommended inference configurations in an output txt file would make sense?

Comment (xiaoyu-work) [reply docs/source/features/packaging_output_models.md:25]: We will still generate an `inference_config.json` file for the recommended inference configuration. Do you think we should still have a sample code for how to use this?

Comment (devang-ml) [reply docs/source/features/packaging_output_models.md:25]: Then it should be sufficient.

**Review by trajepl — APPROVED** [review]

**Pull request merged**